_src/handle-issue-comment.js

```diff
@@ -0,0 +1,112 @@
+const comments = require("./utils/comments.js");
+
+function claim(context, message) {
+  const api = context.github;
+  const labels = context.payload.issue.labels.map(l => l.name);
+
+  api.issues.removeLabel(context.issue({ name: "available" }));
+  api.issues.addLabels(context.issue({ labels: ["in progress"] }));
+  api.issues.createComment(context.issue({ body: message }));
+}
+
+function unclaim(context) {
+  const api = context.github;
+  const labels = context.payload.issue.labels.map(l => l.name);
+  const assignees = context.payload.issue.assignees;
+
+  api.issues.removeLabel(context.issue({ name: "in progress" }));
+
+  if (assignees.length === 0) {
+    api.issues.addLabels(context.issue({ labels: ["available"] }));
+  }
+  api.issues.createComment(context.issue({ body: comments.unclaim }));
+}
+
+function assignAuthor(context) {
```

Comment:
Would rename this to addAssignee as it seems to be a more fitting name regarding action and naming conventions